public/styles/noscript.css

```
@@ -103,9 +181,20 @@ html {
 
   height: 100%;
   width: 100%;
-  font-family: 'HelsinkiGrotesk', Arial, -apple-system, BlinkMacSystemFont,
-    'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
-    'Droid Sans', 'Helvetica Neue', sans-serif;
+  font-family:
+    'HelsinkiGrotesk',
```

Stylelint (17.12.0)

[error] Expected no quotes around "HelsinkiGrotesk" (font-family-name-quotes)

(font-family-name-quotes)